theano/gpuarray/extra_ops.py

```diff
@@ -450,12 +450,13 @@ def c_support_code_struct(self, node, nodename):
         """ % locals()
         return super(GpuCumsum, self).c_support_code_struct(node, nodename) + code
 
-
+@register_opt('fast_compile')
 @op_lifter([CumsumOp])
-def use_gpu_cumsumop(node, ctx_name):
-    if node.inputs[0].dtype == 'float32':
-        axis = node.op.axis
-        x = node.inputs[0]
+@register_opt2([CumsumOp], 'fast_compile')
```

Comment:
If i try to use the decorator the other way, like this, 
`register_opt('fast_compile')(register_opt2([CumsumOp], 'fast_compile')(use_gpu_cumsumop))`
i get this error, 
```
.ERROR (theano.gof.opt): SeqOptimizer apply <theano.gpuarray.opt.GraphToGPU object at 0x11c4dbb90>
ERROR (theano.gof.opt): Traceback:
ERROR (theano.gof.opt): Traceback (most recent call last):
  File "/Users/Ramana/projects/theano/Theano/theano/gof/opt.py", line 235, in apply
    sub_prof = optimizer.optimize(fgraph)
  File "/Users/Ramana/projects/theano/Theano/theano/gof/opt.py", line 90, in optimize
    ret = self.apply(fgraph, *args, **kwargs)
  File "/Users/Ramana/projects/theano/Theano/theano/gpuarray/opt.py", line 318, in apply
    out_clients)
TypeError: 'FromFunctionLocalOptimizer' object is not callable
```